src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/RadioButton_themeresources.xaml

```diff
@@ -21,21 +26,21 @@
             <StaticResource x:Key="RadioButtonBorderBrushDisabled" ResourceKey="ControlFillColorTransparentBrush" />
 
             <StaticResource x:Key="RadioButtonOuterEllipseStroke" ResourceKey="ControlStrongStrokeColorDefaultBrush" />
-            <StaticResource x:Key="RadioButtonOuterEllipseStrokePointerOver" ResourceKey="ControlStrongStrokeColorDefault" />
-            <StaticResource x:Key="RadioButtonOuterEllipseStrokePressed" ResourceKey="ControlStrongStrokeColorDisabled" />
-            <StaticResource x:Key="RadioButtonOuterEllipseStrokeDisabled" ResourceKey="ControlStrongStrokeColorDisabled" />
+            <!-- <StaticResource x:Key="RadioButtonOuterEllipseStrokePointerOver" ResourceKey="ControlStrongStrokeColorDefault" /> -->
```

Comment:
Could you keep the resources uncommented? Even if they are not used, an app could reference them directly (for whatever unrelated reason), so it would be safer to keep them in place.

Reply:
I can't. The same key re-introduced below on L58, but as a Brush instead of a Color.
This could be a breaking chance, if an app were to uses these resources explicitly, and expecting them to be of a `Color`.

Reply:
What if the newly introduced resources used different key (like "UnoRadioButtonOuterEllipseStrokePointerOver") for example (just for the purpose of avoiding the need to uncomment these resources).